File: results/ChatGPT/javascript_React/sortlines.js
import React, { useState } from 'react';

function App() {
  const [inputText, setInputText] = useState('');
  const [outputText, setOutputText] = useState('');

  function processInput() {
    const lines = inputText.split('\n');
    const sortedLines = [...new Set(lines)].sort();
    const result = sortedLines.join('\n');
    setOutputText(result);
  }

  return (
    <div>
      <textarea
        value={inputText}
        onChange={(e) => setInputText(e.target.value)}
        rows={10}
        cols={50}
      />
      <button onClick={processInput}>Process</button>
      <textarea
        value={outputText}
        readOnly
        rows={10}
        cols={50}
      />
    </div>
  );
}

export default App;

File: results/WizardCoder/python_streamlit/sortlines.py
import streamlit as st

def process_text(text):
    lines = text.split('\n')
    lines = sorted(set(lines))
    return '\n'.join(lines)

st.title('Text Processor')

input_text = st.text_area('Enter text here')

if st.button('Process'):
    output_text = process_text(input_text)
    st.text_area('Output', output_text, height=200)